packages/runner/src/storage/cache.ts

```diff
@@ -1265,15 +1271,17 @@ export class Replica {
       cause: null,
     });
 
-    // Temporarily swap the signer to use spaceIdentity for this commit
+    // Temporarily swap the signer to use spaceIdentity (or userIdentity for home space) for this commit
```

Comment:
This signer swap should *only* be used when `this.spaceIdentity` is used -- the `userIdentity` is already set to sign the transaction (the e.g. `remote.as`)